src/ApiService/IntegrationTests/TestLogTracer.cs

```diff
@@ -146,36 +146,30 @@ public class OneFuzzLogger : ILogger {
         } else if (state is EventTelemetry evt) {
             PopulateTags(evt);
             _output.WriteLine($"[Event] {evt}");
-        } else {
-            if ((this as ILogger).IsEnabled(logLevel)) {
-                if (exception is null) {
-                    TraceTelemetry traceTelemetry = new TraceTelemetry(
-                        formatter(state, exception),
-                        OneFuzzLogger.GetSeverityLevel(logLevel));
-
-
-                    traceTelemetry.Context.Operation.Id = Activity.RootId;
-                    traceTelemetry.Context.Operation.ParentId = Activity.SpanId.ToString();
-                    this.PopulateTelemetry(traceTelemetry, state, eventId);
-                    _output.WriteLine($"[Trace] {traceTelemetry}");
```

Comment:
fixing an issue here where the test log would only show the type name of TraceTelemetry